src/routers/user.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
+
+const routes = express.Router();
+
+const { check } = require('express-validator');
```

Comment:
require statements should be at the beginning please move these after requiring express.